jena-arq/Grammar/master.jj

```diff
@@ -100,6 +100,38 @@ import org.apache.jena.sparql.core.Quad ;
 public class CLASS extends PARSERBASE
```

Comment:
tokens.txt is used to produce HTML - hence the additional syntax to indicate inline vs a token rule for each definition.

It is safer to hand edit for minor changes because you risk loosing the additional edits alreayd made.

(If running jj2tokens, send to a temporary file and pick out the new bits and edit in manually)